docs/js/features/odata/use-generated-client.md

```diff
@@ -287,6 +290,33 @@ BusinessPartner.requestBuilder().create(businessPartner);
 
 In the example above we created an instance of BusinessPartner and sent it to the BusinessPartner service in a `POST` request.
 
+#### Deep Create
+
+It is also possible to create an entity together with related entities in a single request:
+
+```ts
+// build a business partner instance with one linked address
+const businessPartner = BusinessPartner.builder()
+  .firstName('John')
+  .lastName('Doe')
+  .businessPartnerCategory('1')
+  .toBusinessPartnerAddress([
+    BusinessPartnerAddress.builder()
+      .country('DE')
+      .postalCode('14469')
+      .cityName('Potsdam')
+      .streetName('Konrad-Zuse-Ring')
+      .houseNumber('10')
+      .build()
+  ])
+  .build();
+
+// execute the create request
+BusinessPartner.requestBuilder()
+  .create(businessPartner)
+  .execute(myDestination);
+```
+
```

Comment:
Optional: I don't remember this myself, but maybe you can find out: When reading this section I think, so what? That is exactly the behavior I would expect ;)
Is there something special that we do? Is the url different than with the normal create? Is there some additional header that we set?
Maybe you can add a comment for future reference.

Reply:
I don't understand, what did you expect? why do you think you need to do something special?
I just wanted to show the reader some code, but I'll add this in a comment anyway

Reply:
I mean - what is the difference between create and deep create?

Reply:
Create Request Builder^:
to create a new entity

Deep Create:
to create an entity together with related entities in a single request

Maybe I'm missing some feature but it seems really simple to use and not worth explaining

Reply:
That's exactly what I mean. It is so obvious, why is it even documented as a feature? I remember that this was not working from the beginning and according to Junjie there are differences in v2 / v4. But it is possible that there is nothing special happening and we just promote the fact that this can also be done. Anyways, I am totally fine with the comment ;)